scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -125,7 +132,8 @@ def get_generated_keys(self) -> List[str]:
 
 class AttackPointIteratorUnrestrictedGenerator(AttackPointIterator):
     """
-    Attack point iterator class that iterates over the unrestricted generator.
+    This exception is raised when the `__len__` function is called
```

Comment:
should I make an extra file for it?

Reply:
In general that is a good idea when a single file is getting larger and larger, but please don't do it in this PR.

Nevertheless the original comment was pointing out that you copy-pasted another comment into the wrong place (here instead of `LengthIsInfiniteException`).